lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
Why is it necessary to place the additional libraries in TomEE/lib folder?
If there are shared libraries they can be placed in the ear/lib as per JAVA EE 7 specification, chapter EE.8.2.1 Bundled Libraries, "library-directory"